pages/linux/wodim.md

```diff
@@ -0,0 +1,24 @@
+# wodim
+
+> Command for record/burning data using a optical drive to CD or DVD.
+> Some usage of wodim can cause destructive actions, such as erasing all the data on a disc.
+
+- Display optical drives avaliable to wodim:
+
+`wodim --devices`
+
+- Display help switches for use with wodim:
+
+`cdrecord`
```

Comment:
I found out that "wodim" in some systems are called "cdrecord" and in others (e.i mine, Linux Mint) it's called both "wodim" and "cdrecord" but no matter what is it called it's "wodim" underneath.
I'll change that in the description so it's clear it's the same command and syntax.